pkg/datapath/linux/ipsec/ipsec_linux.go

```diff
@@ -99,6 +99,55 @@ type oldXfrmStateKey struct {
 	Dst [16]byte
 }
 
+type IPSecParameters struct {
+	// The BootID for the local host, used for bookkeeping.
```

Comment:
Yeah, I just swept that explanation under the rug since it doesn't deal directly with the resulting XFRM state or policy parameters. 

I'll update the wording on this a bit better. By bookkeeping I meant, more of the administrative task of whether or not to create a policy.

Reply:
It's also used for the key derivation.

Reply:
Fixed with latest push